src/applications/search/containers/SearchApp.jsx

```diff
@@ -319,6 +330,18 @@ class SearchApp extends React.Component {
     return [];
   };
 
+  handleInputChange = e => {
+    this.setState({
+      userInput: e.target.value,
+    });
+  };
+
+  fetchInputValue = input => {
```

Comment:
Was this created to handle something we weren't addressing already, like copying/pasting input?

Reply:
This was created to render the correct error message based on input value length, fetching the value from the child component in SearchDropdownComponent